docs/api/client/hoodie.account.rst

```diff
@@ -20,11 +20,11 @@ Example
 
 .. code:: js
 
-    if (hoodie.account.isSignedIn()) {
-    renderWelcome(hoodie.account)
-    }
+    hoodie.account.get('username').then(function (username) {
+      renderWelcome(username);
```

Comment:
Or should i use `hoodie.account.get('username').then(renderWelcome).catch(redirectToHome)`?
The result would be the same, but you can't recognize what parameter is expected.